app/src/main/cpp/skyline/services/codec/IHardwareOpusDecoderManager.h

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <services/serviceman.h>
+
+namespace skyline::service::codec {
+    /**
+     * @brief Struct used to initialize a multi stream decoder
+     * @see opus_multistream_decoder_init()
+     */
+    struct MultiStreamParameters {
+        i32 sampleRate;
+        i32 channelCount;
+        i32 streams; //!< Number of streams
+        i32 coupledStreams; //!< Number of stereo streams
```

Comment:
Change `streams` to `streamCount` and `coupledStreams` to `coupledStreamCount` (or `stereoStreamCount`) and remove the comments.